odoo/tools/misc.py

```diff
@@ -488,6 +488,26 @@ def human_size(sz):
         i += 1
     return "%0.2f %s" % (s, units[i])
 
+
+def human_number(number):
+    """
+    Return the number in a human readable format
+    """
+    sign = '-' if number < 0 else ''
+    number = abs(number)
+    units = [
+        (10 ** 12, 'T'),
+        (10 ** 9, 'G'),
+        (10 ** 6, 'M'),
+        (10 ** 3, 'K'),
```

Comment:
```suggestion
        (10 ** 3, 'k'),
```
cf https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes